bluesky/plans.py

```diff
@@ -2565,6 +2565,8 @@ def adaptive_scan(detectors, target_field, motor, start, stop,
     --------
     :func:`bluesky.plans.relative_adaptive_scan`
     """
+    assert 0 < min_step < max_step, "min_step and max_step must meet condition of max_step > min_step > 0"
```

Comment:
Would this better as a `ValueError` rather than an `AssertionError` since it is triggered by an illegal user-provided value?

Reply:
sure, looks like assert is most often used in test.

Reply:
CPython reserves the right to disregard assert statements at run time (for speed) so they are not good for runtime checks.